R/ospsuite-env.R

```diff
@@ -54,6 +54,9 @@ ospsuiteEnv$isPKSimLoaded <- FALSE
 # NetTask `DimensionTask` cached for performance benefits. Created the first time it is requested.
 ospsuiteEnv$dimensionTask <- NULL
 
+# Small value added to zero when calculating log
+ospsuiteEnv$LOG_SAFE_EPSILON <- 1e-20
```

Comment:
Using the same value as in Core

Reply:
The plan it to use the one from cor eventually right? (the logsafe function)